src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -48,6 +48,23 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     */
+    public function testInvalidCwd()
+    {
+        // Check that it works fine if the CWD exists
+        $cmd = new Process('echo test > testing1.txt', __DIR__);
+        $this->assertEquals(0, $cmd->run());
+
+        $cmd = new Process('echo test > testing2.txt', __DIR__.'/notfound/');
+        $this->assertEquals(0, $cmd->run());
```

Comment:
this fails on windows, where suppress_error does make it fallback to getcwd (but fail instead)

Reply:
How should I handle this?

I cannot test on windows, because I do not have a device to test it with.